internal/testkeys/testkeys.go

```diff
@@ -431,31 +439,56 @@ func RandomSeparator(dst, a, b []byte, suffix int64, maxLength int, rng *rand.Ra
 	diff := bpIdx - apIdx
 	generatedIdx := bpIdx
 	if diff > 0 {
+		var start int64 = diff + 1
```

Comment:
nit: can we use a different name for `start`, maybe `delta` or so? it took me a while to understand that `base` is what I thought `start` would be, off the name.